website/docs/src/content/quickstart.mdx

```diff
@@ -341,10 +341,233 @@ const stream = await workflow.run({ topic: "latest quantum computing chips" });
+
+```bash
+# Start the development server
+gensx start src/workflows.tsx
+```
+
+The development server provides several key features:
+
+- **Hot reloading**: Changes to your code are automatically detected and recompiled
+- **API endpoints**: Each workflow is exposed as a REST endpoint
+- **Swagger UI**: Interactive documentation for your workflows at `http://localhost:1337/swagger-ui`
+- **Local storage**: Built-in support for blob storage and databases
+
+You'll see something like this when you start the server:
+
+```bash
+🔍 Starting GenSX Dev Server...
+ℹ Using project name from gensx.yaml: my-project
+ℹ Starting development server...
+✔ Compilation completed
+✔ Generating schema
+
+📋 Available workflows:
+- ResearchAndWriteWorkflow: http://localhost:1337/local/my-project/default/workflows/ResearchAndWriteWorkflow/run
+
+✅ Server is running. Press Ctrl+C to stop.
+```
+
+You can now test your workflow by sending requests to the provided URL using any HTTP client, or using the built-in Swagger UI at `http://localhost:1337/swagger-ui`.
+
+## Deploying your project to GenSX Cloud
+
+GenSX Cloud provides serverless deployment with zero configuration:
+
+```bash
+# Deploy your project to GenSX Cloud
+gensx deploy src/workflows.tsx -ev OPENAI_API_KEY
+```
+
+This command:
+
+1. Builds your TypeScript code for production
+2. Bundles all dependencies
+3. Uploads the package to GenSX Cloud
+4. Creates REST API endpoints for each workflow
+5. Configures serverless infrastructure
+
+For production deployments, you can target a specific environment:
+
+```bash
+# Deploy to production environment
+gensx deploy src/index.ts --environment production
+```
+
+### Running a workflow from the CLI
+
+Once deployed, you can execute your workflows directly from the command line:
+
+```bash
+# Run a workflow synchronously
+gensx run ResearchAndWriteWorkflow --input '{"topic":"quantum computing"}'
+
+# Save the output to a file
+gensx run ResearchAndWriteWorkflow --input '{"topic":"quantum computing"}' --wait --output result.json
+```
+
+The CLI makes it easy to test your workflows and integrate them into scripts or automation.
+
+### Running a workflow from the GenSX console
+
+The GenSX Cloud console provides a visual interface for managing and executing your workflows:
+
+1. Log in to [app.gensx.com](https://app.gensx.com)
+2. Navigate to your project and environment
+3. Select the workflow you want to run
+4. Click the "Run" button and enter your input
+5. View the results directly in the console
+
+![Running a workflow in the console](/cloud/console-playground.png)
+
+The console also provides API documentation and code snippets for your workflows as well as execution history and tracing for all previous workflow runs.
+
+## Adding persistence with storage
+
+GenSX Cloud provides built-in storage solutions with zero configuration:
```

Comment:
```suggestion
Now that you've deployed your first workflow, you can add in storage so you can build more sophisticated workflows. GenSX Cloud provides built-in storage solutions with no setup required:
```

Reply:
Probably also worth quickly stating "GenSX offers three types of storage: blob storage, sql database storage and, full text and vector search storage."